addons/test_mail/static/tests/properties_field_tests.js

```diff
@@ -0,0 +1,75 @@
+/** @odoo-module **/
+
+import { start, startServer, click } from "@mail/../tests/helpers/test_utils";
+
+QUnit.module("properties field");
+
+/**
+ * Open the chatter when clicking on an avatar many2one / many2many properties.
+ */
+async function testPropertiesOpenChatter(assert, propertyType) {
+    const pyEnv = await startServer();
+    const view = `
+        <form string="Simple">
```

Comment:
string='Form With Avatar Users' ? Just to find something related to the test (not that it changes anything, but hey, have to find stuff to say)